### PR TITLE
Use cached XML in IO XML export job, refs #12998

### PR DESCRIPTION
**Pull request by mcantelon**
Make use of XML cache files when exporting XML via the information
object XML export job.

### CODE REVIEW

File: lib/job/arInformationObjectXmlExportJob.class.php [hunk 1]
```diff
@@ -102,6 +102,8 @@ protected function exportResults($path)
     $search = QubitSearch::getInstance()->index->getType('QubitInformationObject')->createSearch($this->search->getQuery(false, false));
 
     // Scroll through results then iterate through resulting IDs
```
Comment (sbreker) [inline lib/job/arInformationObjectXmlExportJob.class.php]: Nitpick: Comment should be moved down to the foreach line.

File: lib/job/arInformationObjectXmlExportJob.class.php [hunk 2]
```diff
@@ -136,26 +138,36 @@ protected function exportResults($path)
...
+  protected function exportResource($resource, $path, $useCacheFiles)
   {
-    try
-    {
-      // Print warnings/notices here too, as they are often important.
-      $errLevel = error_reporting(E_ALL);
-
-      $rawXml = exportBulkBaseTask::captureResourceExportTemplateOutput($resource, $this->params['format'], $this->params);
-      $xml = Qubit::tidyXml($rawXml);
+    $cachedXmlFilePath = QubitInformationObjectXmlCache::resourceExportFilePath($resource, $this->params['format']);
```
Comment (sbreker) [inline lib/job/arInformationObjectXmlExportJob.class.php]: Does this need to be done if $useCacheFiles is False?

**Review by sbreker — COMMENTED** [review]
Looks good @mcantelon! One nitpick and one question.

File: lib/job/arInformationObjectXmlExportJob.class.php (hunk 2 above)
Comment (mcantelon) [reply lib/job/arInformationObjectXmlExportJob.class.php]: Hmm yeah it doesn't. I've redone this logic to avoid this.

**Comment by mcantelon** [comment]
Thanks for the feedback @sbreker ... I've updated the PR!

**Review by sbreker — APPROVED** [review]
Looks great @mcantelon! Thanks for making those couple changes!

**Pull request merged**